sync/class.jetpack-sync-actions.php

```diff
@@ -20,6 +21,9 @@ static function init() {
 
 		if ( self::sync_via_cron_allowed() ) {
 			self::init_sync_cron_jobs();
+		} else if ( wp_next_scheduled( 'jetpack_sync_cron' ) ) {
```

Comment:
this will be executed on every request 😱 I think it should be fast as far as the`cron` option is autoloaded, is it?